pkg/accountmanager/poller.go

```diff
@@ -126,8 +126,8 @@ func (p *accountPoller) addOrUpdateSelector(selector *crdv1alpha1.CloudEntitySel
 		}
 	}
 
-	// Populate selector specific fields in the accPoller created by CPA, needed for setting owner reference in VM CR.
-	p.selector = selector.DeepCopy()
+	// Store selector to filter cloud resources based on selector.
+	p.selector = selector
```

Comment:
continue using Deepcopy. This way, we release buffer of selector from reconciler

Reply:
Updated the account config structure to use deepcopy. Poller can operate on the selector stored in the account config.